<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>生成树</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menuLl">
    <property name="title">
     <string>场景</string>
    </property>
    <addaction name="actionLl1"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>模型</string>
    </property>
    <addaction name="action_7"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuLl"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>生成树形</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>生成树形纹理</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>生成树叶</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>生成树叶纹理</string>
   </property>
  </action>
  <action name="actionLl1">
   <property name="text">
    <string>场景设计模式</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>导出场景</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>生成场景</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>导入模型</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
